homeassistant/components/sensor/gtfs.py

```diff
@@ -219,11 +223,16 @@ def icon(self):
         """Icon to use in the frontend, if any."""
         return ICON
 
+    @property
```

Comment:
Like I said, it's the first time I ever wrote Python. I'm not sure what's `@property` meant for. I expected the offset to show up in the sensor attributes (on the web UI), but it doesn't. Is this property mandatory?